etc/run-clang-tidy-cached.cc

```diff
@@ -321,9 +322,10 @@ class ClangTidyRunner
         const std::string tmp_out = final_out.string() + uniquifier + ".tmp";
         // Putting the file to clang-tidy early in the command line so that
         // it is easy to find with `ps` or `top`.
-        const std::string command = clang_tidy_ + " '" + work.first.string()
-                                    + "'" + clang_tidy_args_ + "> '" + tmp_out
-                                    + "' 2>/dev/null";
+        const std::string command
+            = clang_tidy_ + " -p " + std::string(compile_json_str) + " '"
```

Comment:
The option description of `clang-tidy` looks like it wants the directory, not the path to the config file; so did you double-check that you get the same output ?

Reply:
I checked it was the same summary